### PR TITLE
Translations as CSV files

### PR DESCRIPTION
**Pull request by rikukissa**

### CODE REVIEW

**Comment by euanmillar** [comment]
This needs cherry-picked on top of release-v1.4.1 or perhaps release-v1.4.2

**Review by Zangetsu101 — COMMENTED** [review]
I would say to bring back the "validate-translations" script & amend the "sort-translations" one before merging this in

File: src/api/content/service.ts [hunk 1]
```diff
@@ -21,19 +22,33 @@ export interface ILanguage {
...
+
+  return languages.map((lang) => {
+    const messages: IMessageIdentifier = {}
+    csvData.forEach((row) => {
+      messages[row.id] = row[lang]
+    })
+
+    return {
+      lang,
+      displayName: lang,
```
Comment (Zangetsu101) [inline src/api/content/service.ts]: I think we use the display name for the "LanguageSelect" dropdowns. So they need to be you know proper labels

**Review by Zangetsu101 — APPROVED** [review]
Looks good @rikukissa apart from the display label I mentioned

**Review by Zangetsu101 — APPROVED** [review]

**Pull request merged**